packages/liveblocks-yjs/src/index.ts

```diff
@@ -43,7 +43,7 @@ export class LiveblocksYjsProvider<
   extends Observable<unknown>
   implements IYjsProvider
 {
-  private room: Room<P, S, U, E, M>;
+  public room: Room<P, S, U, E, M>;
```

Comment:
Not sure if it's really needed to make this a public field. My understanding is that `room` is only ever assigned in the constructor, so effectively this could remain a private property and can even be marked as a `private readonly` property to make that more obvious that its value won't change over the lifetime of the YjsProvider (and we can do the same for the other readonly properties on this class, like `rootDoc` etc).

This matters, because… 1/2